LongWarOfTheChosen/Src/LW_Toolbox_Integrated/Classes/XComGameState_Unit_LWRandomizedStats.uc

```diff
@@ -217,6 +247,14 @@ function ApplyDeltas(XComGameState_Unit Unit, bool Apply)
 		}
 		//`LOG("RandomStats (" $ string(ECharStatType(idx)) $ ") : Old=" $ OldValue $ ", New=" $ NewValue,, 'LW_Toolbox');
 	}
+	if (Apply)
+	{
+		ChangeUnitsCombatIntelligence(Unit, CharacterComIntDelta);
+	}
+	else
+	{
+		ChangeUnitsCombatIntelligence(Unit, -CharacterComIntDelta);
+	}
```

Comment:
This is the same mechanism as changing comint through covert actions, it's pretty retroactive. if someone disables NCE after spending the AP then just let him cheat.

Reply:
How is that comparable? The covert actions can only *increase* the combat intelligence. That can't be rolled back. Speaking of which, does this work properly with the covert action? If the player switches between NCE and no NCE, does the bonus from the CA still apply?

Reply:
if the player did CA first and then switched NCE on, it will still be +2 levels, but it it was +1 which got increased to +2 with CA, then after switching off it should still apply.